total_tolles_ferleihsystem/config.py

```diff
@@ -13,10 +13,67 @@ class Config(object):
     SQLALCHEMY_DATABASE_URI = 'sqlite://:memory:'
     SQLALCHEMY_TRACK_MODIFICATIONS = False
     WEBPACK_MANIFEST_PATH = './build/manifest.json'
-    LOG_PATH = '/tmp'
-    LOG_FORMAT = '%(asctime)s [%(levelname)s] [%(name)-16s] %(message)s <%(module)s, \
-                 %(funcName)s, %(lineno)s; %(pathname)s>'
-    AUTH_LOG_FORMAT = '%(asctime)s [%(levelname)s] %(message)s'
+    LOG_LEVEL = logging.INFO
+    LOGGING = {
+        'version': 1,
+        'formatters': {
+            'default': {
+                'format': '%(asctime)s [%(levelname)s] [%(name)-16s] %(message)s <%(module)s, \
+                 %(funcName)s, %(lineno)s; %(pathname)s>',
+            },
+            'auth': {
+                'format': '[%(asctime)s] %(levelname)s in %(module)s: %(message)s',
+            },
+            'lending': {
+                'format': '%(asctime)s [%(levelname)s] %(message)s',
+            }
+        },
+        'handlers': {
+            'default': {
```

Comment:
Missing Task Logger?

Reply:
Task logging uses the default handler. (As before)